presto-main/src/main/java/io/prestosql/execution/scheduler/NodeSchedulerExporter.java

```diff
@@ -35,12 +35,11 @@
     private final List<MBeanExport> mbeanExports = new ArrayList<>();
 
     @Inject
-    public NodeSchedulerExporter(NodeScheduler nodeScheduler, MBeanExporter exporter)
+    public NodeSchedulerExporter(TopologyAwareNodeSelectorFactory nodeSelectorFactory, MBeanExporter exporter)
```

Comment:
Just noticed that we didn't export any information on the from `UniformNodeSelector`. I think it is good to export the number of co-located and redistribute split for observability.

Reply:
Can you file an issue or put up a PR for that?